src/interface_r/DESCRIPTION

```diff
@@ -3,7 +3,9 @@ Type: Package
 Title: R Interface to 'H2O4GPU'
 Version: 0.2.0
 Authors@R: c(
-  person("Yuan", "Tang", role = c("aut", "cre"), email = "terry@h2o.ai"),
+  person("Yuan", "Tang", role = c("aut", "cre"),
+         email = "terrytangyuan@gmail.com",
+         comment = c(ORCID = "0000-0001-5243-233X")),
```

Comment:
This is just an unique identifier for my involvement (many people in China have the exact same name as mine)...